packages/data/tests/unit/utils/metric-test.js

```diff
@@ -147,4 +148,47 @@ module('Unit - Utils - Metrics Utils', function() {
       'Parser throws an error for corrupted metric'
     );
   });
+
+  test('map column attributes to object', function(assert) {
+    assert.expect(6);
+
+    assert.deepEqual(
+      mapColumnAttributes({ name: 'base' }),
+      { metric: 'base' },
```

Comment:
What do you think of always including a parameters property to the returned metric object? I think it would be good to have a consistent shape for the metric object whether it has parameters or not.